PE/__init__.py

```diff
@@ -747,7 +754,7 @@ def readStringAtRva(self, rva, maxsize=None):
         return ret
 
     def parseImports(self):
-        self.imports = []
+        [ self.imports.pop() for i in range(len(self.imports)) ]
```

Comment:
So because there's a recursion error happening in the CI related to this, I would revert this line back to `self.imports = []` (and similarly for line 767), and then see my other comment on line 820 about making parseImportTable just return the list of imports (and not have it try to modify any parameters or class variables)

Reply:
Sounds good, I made the change.